Gemfile

```diff
@@ -10,7 +10,7 @@ if branch != 'master' && branch < 'v2.0'
 end
 
 gem 'pg', '~> 0.21'
-gem 'mysql2'
+gem 'mysql2', '~> 0.4.10'
```

Comment:
It might be worth bumping this to `~> 0.5.0`, as that's what in Solidus at the moment. However, I don't think that's a blocker for merging.